examples/js/exporters/GLTFExporter.js

```diff
@@ -370,14 +370,20 @@ THREE.GLTFExporter.prototype = {
 
 			var gltfImage = {};
 
+			var canvas = document.createElement( 'canvas' );
```

Comment:
What about creating just one canvas and reuse it on every image by resizing it?

Reply:
Done.